include/quic/btstream.hpp

```diff
@@ -10,7 +10,7 @@ namespace oxen::quic
     using time_point = std::chrono::steady_clock::time_point;
 
     // timeout is used for sent requests awaiting responses
-    inline constexpr std::chrono::seconds TIMEOUT{10};
+    inline constexpr std::chrono::milliseconds TIMEOUT{10s};
```

Comment:
No need to change this; a `seconds` is implicitly convertible to a `milliseconds` (or other more-precise types).

Reply:
Also perhaps:
```suggestion
    inline constexpr auto TIMEOUT = 10s;
```